src/css/application.css

```diff
@@ -113,13 +115,14 @@ body {
 /** @define top-bar */
 
 .top-bar {
+  align-items: center;
   background-color: var(--color-green);
+  color: rgba(0, 0, 0, 0.75);
   display: flex;
   flex: 0 0 auto;
-  font-size: 2vh;
+  font-size: 1.7vh;
   font-weight: bold;
-  align-items: center;
-  padding: 0 1em;
+  padding: 1vh;
```

Comment:
Shouldn’t this be `1vw 1vh`?

Reply:
Actually it works just as well in ems. Changed to ems.